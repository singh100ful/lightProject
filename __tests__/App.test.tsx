/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../src/App';

// Note: import explicitly to use the types shiped with jest.
import {it} from '@jest/globals';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('App', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
