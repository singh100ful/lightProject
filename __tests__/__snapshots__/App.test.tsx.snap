// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#F3F3F3",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "flex": 1,
        "padding": 23.125,
      }
    }
  >
    <Text>
      LoginScreen
    </Text>
  </View>
</RCTSafeAreaView>
`;
